contracts/src/BlobstreamX.sol

```diff
@@ -41,29 +41,28 @@ contract BlobstreamX is
     /// @notice Next header function id.
     bytes32 public nextHeaderFunctionId;
 
+    struct InitParameters {
```

Comment:
nit: IMO using struct doesn't really help here since its not being passed around

Reply:
stack too deep